packages/core/tests/v1-indy-proofs.test.ts

```diff
@@ -252,6 +252,54 @@ describe('Present Proof', () => {
       connectionId: expect.any(String),
       state: ProofState.Done,
     })
+
+    const proposalMessage = await aliceAgent.proofs.findProposalMessage(aliceProofRecord.id)
+    const requestMessage = await aliceAgent.proofs.findRequestMessage(aliceProofRecord.id)
+    const presentationMessage = await aliceAgent.proofs.findPresentationMessage(aliceProofRecord.id)
+
+    expect(proposalMessage).toBeInstanceOf(V1ProposePresentationMessage)
+    expect(requestMessage).toBeInstanceOf(V1RequestPresentationMessage)
+    expect(presentationMessage).toBeInstanceOf(V1PresentationMessage)
+
+    const formatData = await aliceAgent.proofs.getFormatData(aliceProofRecord.id)
+
+    expect(formatData).toMatchObject({
+      proposal: {
+        indy: {
+          requestedAttributes: [
+            {
+              name: 'name',
+              value: 'John',
+              referent: '0',
+            },
+            {
+              name: 'image_0',
+            },
+          ],
+          requestedPredicates: [
+            {
+              name: 'age',
+              predicate: '>=',
+              threshold: 50,
+            },
+          ],
+        },
+      },
+      request: {
+        indy: {
+          name: 'Proof Request',
+          version: '1.0',
+          nonce: expect.any(String),
```

Comment:
This doesn't include the whole proof request. I think it would be good to check that here, to make sure the casing is correct (it should be snake_case here as that's what indy uses natively)